MODULE.bazel

```diff
@@ -1,8 +1,8 @@
 module(
     name = "rules_jvm_external",
+    version = "6.1",
```

Comment:
I feel like I've seen this line being rearranged a ton.. 🤔

Reply:
Sigh, reverted the spurious formatting changes :)